va/va_fei.h

```diff
@@ -61,6 +61,102 @@ extern "C" {
 
 /**@}*/
 
+/** \brief Attribute value for VAConfigAttribStats */
+typedef union _VAConfigAttribValStats {
+    struct {
+        /** \brief Max number of past reference frames that are supported. */
+        uint32_t    max_num_past_references   : 4;
+        /** \brief Max number of future reference frames that are supported. */
+        uint32_t    max_num_future_references : 4;
+        /** \brief Number of output surfaces that are supported */
+        uint32_t    num_outputs               : 3;
+        /** \brief Interlaced content is supported */
+        uint32_t    interlaced                : 1;
+        uint32_t    reserved                  : 20;
+    } bits;
+    uint32_t value;
+} VAConfigAttribValStats;
+
+typedef struct _VAPictureFEI
+{
+    VASurfaceID picture_id;
+    /*
+     * see flags below.
+     */
+    uint32_t flags;
+} VAPictureFEI;
+/* flags in VAPictureFEI could be one of the following */
+#define VA_PICTURE_FEI_INVALID                   0x00000001
+#define VA_PICTURE_FEI_PROGRESSIVE               0x00000000
+#define VA_PICTURE_FEI_TOP_FIELD                 0x00000002
+#define VA_PICTURE_FEI_BOTTOM_FIELD              0x00000004
+/** \brief picutre surface content updated indicator.
+ * The picture surface content is updated, it means temporary buffer like downscaled pixel data in driver
+ * internal needs be forced freshing
+ **/
+#define VA_PICTURE_FEI_CONTENT_UPDATED           0x00000010
+
```

Comment:
Similar to the previous comment, why  VA_PICTURE_FEI_*  came as preferred name in favor of VA_PICTURE_STATS_*  ?

Reply:
Yes, VA_PICTURE_STATS_ may be more accurate since it's only used for VAEntrypointStats